packages/offix-datasync-codegen/tests/__snapshots__/DataSyncClientPlugin.test.ts.snap

```diff
@@ -2,74 +2,95 @@
 
 exports[`Test datasync config 1`] = `
 "import { DataStore } from 'offix-datastore';
-import schema from './schema.json';
+import schema from './schema';
 
-const datastore = new DataStore({
+export const datastore = new DataStore({
     dbName: \\"offix-datasync\\",
     clientConfig: {
       url: \\"http://localhost:4000/graphql\\",
       wsUrl: \\"ws://localhost:4000/graphql\\",
     }
 });
 
-datastore.createModel(schema.Note);
-datastore.createModel(schema.Comment);
+export const NoteModel = datastore.createModel(schema.Note);
+export const CommentModel = datastore.createModel(schema.Comment);
 
 datastore.init();
 "
 `;
 
 exports[`Test json schema 1`] = `
-Array [
-  Object {
-    "json": Object {
-      "name": "Note",
+Object {
+  "json": Object {
+    "Comment": Object {
+      "name": "Comment",
+      "primaryKey": "id",
       "properties": Object {
-        "comments": Object {
-          "isRequired": true,
-          "type": "string[]",
-        },
         "description": Object {
           "isRequired": true,
+          "key": "description",
           "type": "string",
         },
         "id": Object {
+          "index": true,
           "isRequired": true,
+          "key": "id",
+          "primary": true,
+          "type": "string",
+        },
+        "noteComment": Object {
+          "isRequired": true,
+          "key": "noteComment",
           "type": "string",
         },
         "title": Object {
           "isRequired": true,
+          "key": "title",
           "type": "string",
         },
       },
       "type": "object",
       "version": 1,
     },
-  },
-  Object {
-    "json": Object {
-      "name": "Comment",
+    "Note": Object {
+      "name": "Note",
+      "primaryKey": "id",
       "properties": Object {
-        "description": Object {
+        "comments": Object {
           "isRequired": true,
-          "type": "string",
+          "key": "comments",
+          "type": "string[]",
         },
-        "id": Object {
+        "description": Object {
           "isRequired": true,
+          "key": "description",
           "type": "string",
         },
-        "noteComment": Object {
+        "id": Object {
+          "index": true,
           "isRequired": true,
+          "key": "id",
+          "primary": true,
           "type": "string",
         },
         "title": Object {
           "isRequired": true,
+          "key": "title",
           "type": "string",
         },
       },
       "type": "object",
       "version": 1,
     },
   },
-]
+  "schemaExport": "import { DataSyncJsonSchema } from \\"offix-datastore\\";
+import schema from \\"./schema.json\\";
+
+type Schema<T = any> = {
+    [P in keyof T]: DataSyncJsonSchema<any>
+};
+
+export schema as Schema;
```

Comment:
I think this might need to be `export default schema ...`

Reply:
Yes. I'm changing this currently

Reply:
@Eunovo cool your solution was better because we get a named export.

Reply:
I needed to explicitly set the type of the export and I didn't know how to do that in default export